annotation/annotation/distribution/main.py

```diff
@@ -72,6 +77,8 @@
 )
 from annotation.tasks import create_tasks as create_db_tasks
 
+dotenv.load_dotenv(dotenv.find_dotenv())
+SPLIT_MULTIPAGE_DOC = os.getenv("SPLIT_MULTIPAGE_DOC", "false")
```

Comment:
I would use bool variable instead of str
like 
`SPLIT_MULTIPAGE_DOC = True if os.getenv("SPLIT_MULTIPAGE_DOC", "false") == "true" else False`